stdlib/universe/die_test.go

```diff
@@ -0,0 +1,45 @@
+package universe_test
+
+import (
+	"context"
+	"github.com/influxdata/flux"
+	"testing"
+
+	"github.com/apache/arrow/go/arrow/array"
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/dependencies/dependenciestest"
+	"github.com/influxdata/flux/stdlib/universe"
+	"github.com/influxdata/flux/values"
+)
+
+func TestDie(t *testing.T) {
+	for _, tc := range []struct {
+		name    string
+		msg string
+		data func() *array.Float64
+		want error
+	}{
+		{
+			name: "foo",
+			msg: "this is an error message",
+			want: &flux.Error{
+				Code: codes.Internal,
+				Msg:  "this is an error message",
+			},
+		},
+	} {
+		t.Run(tc.name, func(t *testing.T) {
+			dieFn := universe.Die()
+			fluxArg := values.NewObjectWithValues(map[string]values.Value{"msg": values.NewString(tc.msg)})
+			_, got := dieFn.Call(dependenciestest.Default().Inject(context.Background()), fluxArg)
+			if got == nil {
+				t.Fatal("this function should produce an error")
+			}
+
+			if !cmp.Equal(tc.want, got) {
+				t.Fatalf("unexpected result -want/+got\n\n%s\n\n", cmp.Diff(tc.want, got))
+			}
+		})
+	}
+}
```

Comment:
@rockstar added a unit test here. Lemme know what you think.

Reply:
I'm not sure what the common pattern for go tests usually is here. It seems awkward to create a list of cases that is one test case long. Maybe just calling `t.Run` directly for your one case, instead of iterating over them, would make things better. Maybe this is idiomatic go, though? I'm definitely not an expert there.

Reply:
No, its not really necessary here. You're totally right to point that out. Remnant from when I thought I might have more cases to test.